### PR TITLE
[FIX] auto creation: link existing timesheets

### PR DESCRIPTION
**Pull request by huguesdk**
[task](https://gestion.coopiteasy.be/web#id=7858&view_type=form&model=project.task)

hr_timesheet_auto_creation: link existing timesheets (account.analytic.line) to newly automatically created timesheet sheets.

### CODE REVIEW

File: hr_timesheet_auto_creation/models/hr_timesheet_sheet.py [hunk 1]
```diff
@@ -35,14 +35,20 @@ def create_employee_timesheet(self):
             lambda x: x.employee_id.id, exists_timesheet_records
         )
         employee_ids = list(set(employee_ids) - set(ignore_employee_ids))
+        sudo_self = self.sudo()
         for employee_id in employee_ids:
-            self.sudo().create(
+            tss = sudo_self.create(
```
Comment (carmenbianca) [inline hr_timesheet_auto_creation/models/hr_timesheet_sheet.py]: I would probably rename this to `sheet` or some such as a more descriptive variable.

**Review by carmenbianca — CHANGES_REQUESTED** [review]

File: hr_timesheet_auto_creation/models/hr_timesheet_sheet.py (hunk 1 above)
Comment (huguesdk) [reply hr_timesheet_auto_creation/models/hr_timesheet_sheet.py]: indeed! `tss` means “timesheet sheet”, but it’s a little cryptic.

**Review by robinkeunen — COMMENTED** [review]
Sorry for the late review. 👍 for me, i'm just commenting to warn about asigning a "sudo record" to a variable. When doing so, one must be careful to only use it where sudo is needed since it bypasses all access right. Do not, for example, return the sudo'ed record as it might wander around the code.

Just a warning, it's fine here.

**Comment by huguesdk** [comment]
good to know, thanks!

**Pull request merged**